docs/content/manual/pre-release/node/node-drain-deletion.md

```diff
@@ -65,3 +65,39 @@ NOTE: We might still 1 or 2 times of the pod's disruption budget message, but wi
 evicting pod longhorn-system/instance-manager-r-xxxxxxxx
 error when evicting pods/"instance-manager-r-xxxxxxxx" -n "longhorn-system" (will retry after 5s): Cannot evict pod as it would violate the pod's disruption budget.
 ```
+
+
+## Stopped replicas on deleted nodes should not be counted as healthy replicas when draining nodes
+
+When draining a node, the node will be set as unscheduled and all pods should be evicted.
+
+But the replica will only be evicted if it has another healthy replica on the running node.
+
+##### Related Issue:
+- https://github.com/longhorn/longhorn/issues/2237
+
+##### Prerequisites:
+ - Start with 2 nodes cluster: Node_1, Node_2
+ - Double check that "Allow Node Drain with the Last Healthy Replica" is unchecked or return **false** with following command:
+    - `kubectl get settings.longhorn.io/allow-node-drain-with-last-healthy-replica -n longhorn-system`
+
+##### Steps:
+1. Create a 5Gi detached volume with 2 replicas.
+2. Stop Node_1 that contains one of the replicas.
+3. Attempts to drain Node_2 that contains remaining replica.
+    ```shell
+    kubectl drain <Node name> --delete-emptydir-data=true --force=true --grace-period=-1 --ignore-daemonsets=true --timeout=<Desired timeout in secs>
```

Comment:
Can you clarify why we need `--timeout` ? Does it make a difference with and without setting this flag?

Reply:
No, there is no different result between with and without parameter `timeout`.

Since after the node_2 is drained in step 3, it keeps giving the message that replica cannot be evicted as mentioned in step 4, I pass the `timeout` to indicate length of waiting time.

I figured I'd remove the argument to avoid confusion.

Reply:
No, there is no different result between with and without parameter `timeout`.

Since after the node_2 is drained in step 3, it keeps giving the message that replica cannot be evicted as mentioned in step 4, I pass the `timeout` to indicate length of waiting time.

I figured I'd remove the argument to avoid confusion.